js/app/modules/card/media.js

```diff
@@ -58,12 +58,12 @@ const Media = new Module.Class({
             attributions.push(this.model.copyright_holder);
         this.set_label_or_hide(this._attribution_label, attributions.map((s) => {
             return s.split('\n')[0];
-        }).join(' - ').toUpperCase());
+        }).join(' - '));
         this._attribution_button.visible = this._attribution_label.visible;
         this._attribution_button.connect('clicked', this._show_license_in_external_viewer.bind(this));
-
-        this._separator.visible = this._attribution_label.visible &&
-                                  this._caption_label.visible;
+        if (this._caption_label.visible && this._attribution_label.visible) {
```

Comment:
@ptomato  I agree with your comment about BEM naming, but this solution seems odd to me. to my knowledge, in CSS, you can't give 2 different classes certain styling depending on if their _container contains_ those 2 classes. I think, rather, you can give 2 classes certain styling depending on if its container acutally has those 2 classes applied to _it_... I think that's what you're suggesting anyway, to add `--caption` and `--attribution` to the parent module, but that's kind of doing the same thing that I suggested, right? But just adding 2 new classes to the container rather than one (which i named `CaptionAndAttrVisible` or something. 

I agree that should be done in CSS, but I didn't think it was possible.